jvm/src/main/scala/eval/directive/OpDirectives.scala

```diff
@@ -14,11 +14,16 @@ import Validated._
 import geotrellis.vector._
 import geotrellis.raster.{Tile, isData}
 
+import scala.reflect.ClassTag
 import scala.concurrent.duration._
 import scala.util.Try
 
 
 object OpDirectives {
+
+  private def asInstanceOfOption[T: ClassTag](o: Any): Option[T] = 
```

Comment:
The main reasons for this are 1. we lost `as[G <: Geometry]` from core when we moved to fully supporting `jts` geometries and 2. to avoid the runtime errors which are otherwise unavoidable when casting. `ClassTag` should get around erasure (kind of) but only because it passes around some context explicitly describing the type which can be checked at runtime.